applications/StructuralMechanicsApplication/structural_mechanics_application_variables.cpp

```diff
@@ -125,8 +125,10 @@ KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(NODAL_DAMPING_RATIO)
 // For explicit central difference scheme
 KRATOS_CREATE_VARIABLE(double, MASS_FACTOR)
 KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(MIDDLE_VELOCITY)
-KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(NODAL_INERTIA)
 KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(MIDDLE_ANGULAR_VELOCITY)
+KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(NODAL_INERTIA)
+KRATOS_CREATE_VARIABLE(double, NODAL_DISPLACEMENT_DAMPING)
+KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(NODAL_ROTATION_DAMPING)
```

Comment:
why is `NODAL_ROTATION_DAMPING` an array and `NODAL_DISPLACEMENT_DAMPING` a double?

Reply:
There is not a mass per each direction. In fact the rotational should be a tensor not an array as @RiccardoRossi says

Reply:
ok
@KlausBSautter to be discussed

Reply:
@philbucher we can discuss this tmr at the chair

Reply:
on monday, I am not there tmr :)